### ISSUE TITLE
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color, in Design screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#852]
**Describe the bug**
The visual presentation of the Components/Images don't have a contrast ratio of at least 3:1 against adjacent color.

**To Reproduce**

1.Install "Color Contrast Analyzer" extension in your browser.
2. Go to Design  (http://localhost:9992/cm/app/?view=design&site=test&path=%2FSites%2Ftest%2F&pathType=page)
3. Scroll to the upper part of the page.
4. Click "Extension" icon given in the chrome browser, in top right. It will open Extension Pop Up.
5. Click "Color Contrast Analyzer", option in pop up. It will open another pop up.
6. Click "Capture Visible Content" from the pop up.
7. Observe that links like "Create new site", "Create New Folder" etc. not highlighted (black and white) by the Color Contrast Analyzer.
8. Perform the same steps after lower part of the page.
9. Observer that the Icons given for moving between pages, in the template section, are also not highlighted.

**Expected behavior**
The visual presentation of the Components/Images should have a contrast ratio of at least 3:1 against adjacent color.

**Additional context**
Navigate all the options available for similar issue and fix.
